canonical-web-and-design/snapcraft.io
# Switch from sass-lint to stylelint

## Done

Removed sass-lint and added stylelint to bring the site inline with our other projects

## Issue / Card

Fixes #3223

## QA

- Pull the branch
- Run the site to install the new dependencies
- Run `dotrun exec yarn lint-scss`
- Make sure it runs and has no linting errors
